content/deploy/community-cloud/deploy-your-app/app-dependencies.md

```diff
@@ -1,29 +1,29 @@
 ---
-title: App dependencies
+title: App dependencies for your Community Cloud app
 slug: /deploy/streamlit-community-cloud/deploy-your-app/app-dependencies
 ---
 
-# App dependencies
+# App dependencies for your Community Cloud app
 
-The main reason that apps fail to build properly is because Streamlit Community Cloud can't find your dependencies! There are two kinds of dependencies your app might have: Python dependencies and external dependencies. Python dependencies are other Python packages (just like Streamlit!) that you `import` into you script. External dependencies are less common, but they include any other software your script needs to function properly. Since Streamlit Community Cloud runs on Linux, these will be Linux dependencies installed with `apt-get` outside the Python environment.
+The main reason that apps fail to build properly is because Streamlit Community Cloud can't find your dependencies! There are two kinds of dependencies your app might have: Python dependencies and external dependencies. Python dependencies are other Python packages (just like Streamlit!) that you `import` into your script. External dependencies are less common, but they include any other software your script needs to function properly. Since Community Cloud runs on Linux, these will be Linux dependencies installed with `apt-get` outside the Python environment.
 
 For your dependencies to be installed correctly, make sure you:
 
 1. Add a [requirements file](#add-python-dependencies) for Python dependencies.
-2. (optional) Add a `packages.txt` file to manage any external dependencies.
+2. (Optional) Add a `packages.txt` file to manage any external dependencies.
 
 <Note>
 
 Python requirements files should be placed either in the root of your repository or in the same
-directory as your Streamlit app.
+directory as your app's entrypoint file.
 
 </Note>
 
 ## Add Python dependencies
 
-With each `import` statement in your script, you are bringing in a Python dependency. You need to tell Streamlit Community Cloud how to install those dependencies through a Python package manager. We recommend using a `requirements.txt` which is based on `pip`.
+With each `import` statement in your script, you are bringing in a Python dependency. You need to tell Community Cloud how to install those dependencies through a Python package manager. We recommend using a `requirements.txt` which is based on `pip`.
```

Comment:
Since `requirements.txt` is canonically "the format `pip` uses," I intended this to provide supporting, additional information. (It's a reminder `requirements.txt` means `pip`, but the information isn't necessary.) So, non-restrictive clause?